xml/containerized-kubernetes.xml

```diff
@@ -62,23 +60,48 @@
  </sect1>
  <sect1 xml:id="kube-prereq">
   <title>Prerequisites</title>
-
   <para>
-   Before you start deploying, you need to have:
+   The minimum requirements and prerequisites to deploy &productname; &productnumber;
+   on top of &casp; 4 &kube; cluster are as follows:
   </para>
-
   <itemizedlist>
    <listitem>
     <para>
-     A running &casp; 4 cluster.
+     A running &casp; 4 cluster. You need to have an account with a
+     &casp; subscription. You can activate a 60-day free evaluation
+     here <link xlink:href="https://www.suse.com/products/caas-platform/download/MkpwEt3Ub98~/?campaign_name=Eval:_CaaSP_4"/>.
     </para>
    </listitem>
    <listitem>
     <para>
-     &casp; 4 worker nodes with a number of extra disks attached as storage for
-     the &ceph; cluster.
+     At least 3 &casp; worker nodes with a number of extra disks attached as
```

Comment:
I suggest:

> Three &casp; worker nodes
&
> We recommend four

Numbers under 10 written out in full is the most common style.

I think "a number of extra disks" is too vague.

How about:

... worker nodes, with at least one additional disk attached to each node as storage...